docs/reST/ref/code_examples/base_script.py

```diff
@@ -1,18 +1,19 @@
 # Example file showing a basic pygame "game loop"
 import pygame
+import sys
 
 # pygame setup
 pygame.init()
 screen = pygame.display.set_mode((1280, 720))
 clock = pygame.time.Clock()
-running = True
 
-while running:
+while True:
```

Comment:
looks like you forgot to change it back to running, since its set to true in line 8 its functionally equivalent but more understandable.